### DISCUSSION TITLE
Question about this picture of wiring

### DISCUSSION THREAD

**Discussion by asotelles** [ysoldak/HeadTracker#55] [General]
https://headtracker.gitbook.io/head-tracker/getting-started/wiring/option-3-bridged/radio-master-tx16s-mk2
On this page of wiki blue wire goes to the tx of the radio instead of rx, is that right?

Other question, does radiomaster pocket has similar internal conector?

**Comment by ysoldak** [comment]
Sorry, @asotelles I have no idea, that's not my wiki and project.
Try and ask at https://github.com/dlktdr/HeadTracker ?

**Reply by asotelles** [reply]
holy smokes, I'm sorry and thank you for reply
